wikipedia_cleanup/data_processing.py

```diff
@@ -158,7 +161,90 @@ def get_data(
             mapped_filters.append(changes_and_filters[1])
     all_changes = itertools.chain.from_iterable(all_changes)
     merge_filter_stats_into(mapped_filters, filters)
-    return pd.DataFrame([change.__dict__ for change in all_changes])
+    res_df = pd.DataFrame([change.__dict__ for change in all_changes])
+    sorted_columns = sorted(list(res_df.columns))
+    return res_df[sorted_columns]
+
+
+def get_data_single(
+    file: Path,
+    data_filters: Optional[List[AbstractDataFilter]] = None,
+) -> pd.DataFrame:
+    if data_filters is None:
+        data_filters = []
+    changes_and_filters = read_and_filter_file(file, data_filters)
```

Comment:
which filters are returned here?

Reply:
I do not understand the question. We are not returning filters here. We just forward an empty list if no filter were provided.

Reply:
its called "changes and filters" so i assumed that you somehow return filters